architectures/firmware/sof-zephyr/rtos_layer/io_drivers/i2s/i2s_driver.rst

```diff
@@ -108,3 +108,15 @@ stream with already running ones.
 
 "Single" I2S links may be synchronized and aggregated by sending I2sSyncData to
 the I2S IO Driver.
+
+Loopback mode
+======================================
+
+In some architectures I2S transmitter and receiver share same physical data pins - 
+the Tx pin of the transmitter is connected to the Rx pin of the receiver. 
```

Comment:
that's not quite right. The connection is at the IP level, not at the physical data pins. We can enable the loopback even if the pins are not connected to anything, e.g. on an Up board or RVP.